onnx2pytorch/operations/range.py

```diff
@@ -0,0 +1,10 @@
+import torch
+from torch import nn
+
+
+class Range(nn.Module):
+    def __init__(self):
```

Comment:
The __init__ is currently redundant since it just calls the super. Could you remove it please?

Reply:
Done!